docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
-- System Ui: systemui=true
-  - If FCA systemui=true, FCA acts as the base app in case of ripple. The background color will be changed to purple, and it will display one more button as "Launch FCA app" to launch FCA as third-party app on Ripple devices.
-- TestContext: testContext=true
-  - If testContext=true, it will add the field context in mocha report generated
-```
+
 
 ### Background
 
-TODO: add details
+URL parameters allow customization of the FCA's behavior, enabling testers to tailor the testing environemt to specific needs.
 
 ### Setup
 
-TODO: add details
+- Use various parameters like 'platform', 'lifecycle_validation','mf','systemui' and more as per the requirment.
+- These parameters modify how the FCA interacts with the environemtn and the test suite.
 
 ### Usage
 
-TODO: add details
+Customize the test execution and reporting based on the provided URL parameters for targeted testing scenarios.
+
+**Parameters usage:**
+- **platform**: name of targeted platform
+- **lifecycle_validation**: When user passes lifecycle_validation=true it blocks the default execution of lifecycle.ready and lifecycle.finished method.
+This will help  to validate lifecycle api's as needed.
+- **mf**: If mf=true is passed in the URL, FCA will connect to MFOS server and when we invoke any api in FCA it will return the response fron MFOS.
+- **systemui**: If FCA systemui=true, FCA acts as the base app in case of ripple. The background color will be changed to purple and it will display one more button as "Launch FCA app" to launch FCA as third-party app on Ripple devices.
+- **testContext**: If testContext=true, it will add the field context in mocha report generated
+- **voiceGuidance** : If FCA voiceGuidance=true app will voice out navigated menu options.
+- **reportingId**: If passed in the URL, it will get appended to the report name and eventually form the reporting URL.
+- **standalone**: Standalone execution is designed for automated testing scenarios, where generating detailed reports and logs is crucial to be within FCA independantly of any other apps or frameworks. Enable standalone execution by setting the standalone=true parameter in the URL example:(https://firecertapp.firecert.com/edge/index.html?standalond=true) or within the intent message example:(intent>data>query>params>"standalone":true).
+
 
 ## App Types
- 
-TODO: add details
 
 ### Background
 
-TODO: add details
+Different app types in FCA denote various categories or classifications of applications that can be tested using the platform. 
 
 ### Setup
 
-TODO: add details
+- Define the app types as part of the configuration before initiating tests.
+- This may involve specifying the type of applocation under test, such as media apps, utility apps, etc.
 
 ### Usage
 
-TODO: add details
+- Select the appropriate app type to ensure that tests are relevant and effective for the specific category of the application.
```

Comment:
This doesn't tell the user how to use app types in FCA.

Reply:
@jbigel in the setup, it's mentioned now that App Type can be sent via the intent message. 
any other addition for this section ?

Reply:
- Define the app types as part of the configuration before initiating tests (AppType can be sent via the intent message) [Remote Execution](https://github.com/rdkcentral/firebolt-certification-app/blob/docs/Execution.md#remotely).

Reply:
How do you define app types `as part of the configuration`. This is the configuration doc so we need to be more explicit on the hows.